src/constants/fonts.scss

```diff
@@ -0,0 +1,102 @@
+@import './colors.scss';
+
+$primary-font : 'Helvetica Neue';
+
+@mixin section-title {
```

Comment:
move to mixins file

but i think its necessary and you can use the group style

  @include mobile-h2;
    font-size: 18px;
    letter-spacing: 1.25px; all of this can be in the group style